src/execution_plan/optimizations/utilize_indices.c

```diff
@@ -195,6 +209,55 @@ bool _simple_predicates(const FT_FilterNode *filter) {
 	return true;
 }
 
+static bool _validateInExpression(AR_ExpNode *exp) {
+	if(!strcasecmp(exp->op.func_name, "in") == 0) return false;
+	assert(exp->op.child_count == 2);
+	AR_ExpNode *list = exp->op.children[1];
+	if(list->type != AR_EXP_OPERAND || list->operand.type != AR_EXP_CONSTANT) return false;
```

Comment:
`if(list->operand.type != AR_EXP_CONSTANT) return false;` ?
why check for `AR_EXP_OPERAND` ?